src/pex/pex.c

```diff
@@ -108,9 +136,13 @@ void pex_set_pin_dir(pex_t* pex, uint8_t pin, pex_bank_t s, pex_dir_t dir) {
     }
 }
 
-// Sets the value of pin 'pin' on bank 's' to value 'v'
-// bank - A (GPIOA) or B (GPIOB)
-// value - HIGH or LOW
+/*
+Sets the value of pin 'pin' on bank 's' to value 'v'
+pex - pointer to the pex device
+pin - the pin to set
+s - the bank the pin is on, A (GPIOA) or B (GPIOB)
+v - the value, must be HIGH or LOW
+*/
 void pex_set_pin(pex_t* pex, uint8_t pin, pex_bank_t s, pex_val_t v) {
```

Comment:
Switch order of pin and s, use a `uint8_t` instead of `pex_val_t`, give s and v better names